docs/designers-developers/developers/tutorials/devenv/readme.md

```diff
@@ -1,6 +1,8 @@
 # Development Environment
```

Comment:
I see that this file is named `readme.md`. I double-checked and it works on GitHub the same way as `README.md` – it displays its content by default when entering the folder:
https://github.com/WordPress/gutenberg/tree/1e0bda1e09d05eab04149171847cc0ee05e99520/docs/designers-developers/developers/tutorials/devenv

So I'm curious why we use capitalized version in all places next to the code :)

Reply:
That's a good question, I don't know why or how it started.